recipes-bsp/sgx/sgx-sdk-cross_2.18.1.bb

```diff
@@ -48,4 +59,3 @@ SYSROOT_DIRS += "${sgxdirprefix}${sgxsdkpath}-cross"
 FILES:${PN}-dev += "${sgxsdkpath}-cross"
 
 CVE_PRODUCT = "software_guard_extensions_sdk"
-
```

Comment:
do not remove the newline at the end of the file

Reply:
Fixed.